source/scripts/components/Snackbar/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+import styles from './snackbar.css';
+
+// TODO: add actionText and onActionClick handler (yes/no, undo, etc..)
```

Comment:
Generally a good practice is to create issues and, if necessary, reference them in code comments.

If these handlers aren't needed in current task, it's better to remove this comment at all since there is no meaning in adding "nice, but not necessary" features.